tests/test_keyring.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+import subprocess
+from pathlib import Path
+
+
+# Update this pubkey ID if the key or its subkeys are updated; see README
+GPG_PUBKEY_ID = "gpg-pubkey-7b22e6a3-60966ad"
+
+# Basic acceptance testing for keyring package.
+# Expects fedora environment.
+
+RPM_QUERY_KEYRING_PACKAGE = ["rpm", "-q", "securedrop-workstation-keyring"]
+RPM_GPG_QUERY_SD_RELEASE_KEY = ["rpm", "-q", GPG_PUBKEY_ID]
+REPOFILE_PATH = "/etc/yum.repos.d/securedrop-workstation-dom0.repo"
+KEYFILE_PATH = "/etc/pki/rpm-gpg/RPM-GPG-KEY-securedrop-workstation"
+
+def test_is_fedora():
+    with open("/etc/os-release") as f:
+        for line in f:
+            if line.startswith("NAME"):
+                return "Fedora" in line.split("=")[-1]
+    return False
+
+def test_is_package_installed():
+    try:
+        # exit code 1 if package is not installed
+        subprocess.check_call(RPM_QUERY_KEYRING_PACKAGE)
+        return True
+    except subprocess.CalledProcessError:
```

Comment:
Consider letting the exception bubble up; it will still cause the test to fail and probably provides a little more output about what failed instead of "asserting False failed".